### ISSUE TITLE
alt seems to stick in certain situation

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#90]
Converted from [SourceForge issue 2932981](http://sourceforge.net/support/tracker.php?aid=2932981), submitted by nobody

I have assigned the Move/Resize key to Right Alt. After I move a window with Right Alt+Left mouse Windows seems to think the Alt key is still depressed. I must press (and release) the other (Left) Alt to solve this.

**Comment by ttencate** [comment]
Submitted by ttencate

This and similar issues are being worked on.

**Comment by mtaran** [comment]
This also happens to me. Steps to reproduce:
1) Hold down alt
2) Hold down right mouse button
3) Lift up alt
4) Lift up mouse button
The result is that alt is stuck down and thus makes it almost impossible to do anything until alt is pressed again cancelling the effect.

**Comment by jvtrigueros** [comment]
I just reproduced the steps from mtaran, and the only thing that I want to note is on the second step. It must be the key you use to re-size. In my case it's the middle mouse button.

**Comment by thinkedem** [comment]
Actually I just did it with the "move" button (that is my LMB).

**Comment by jvtrigueros** [comment]
I built the binary from source last night and I can't seem to repro the problem. It is hard to find out what's actually causing this. Are you able to get consistent reproduction of the bug?

**Comment by ttencate** [comment]
Yes, there have been plenty of structural changes and fixes in HEAD, which might well have fixed the issue.

As it's been over two years now, I don't remember what the blocker was on releasing this as a new version. It could well be that it just required more testing. If so, I'll take care of that next week.

**Comment by jvtrigueros** [comment]
That would be awesome, this program is truly amazing for those of us working in a non-Linux environment. I got the code to see if I could help out with the bug squashing as this was the only bug that caused me frustration, but I could not repro it once I built the code, thank you for looking into it.

**Comment by DootDootThank** [comment]
Bump. Still an issue in 0.3.1.0 on 64 bit windows 7. Followed mtaran's steps to reproduce, 100% reproducible in all windows.
